### DISCUSSION TITLE
http://192.168.1.55:8123/auth/alexamedia/proxy/verify 500 Internal Server Error Server got itself in trouble

### DISCUSSION THREAD

**Discussion by saviothecnic** [alandtse/alexa_media_player#2133] [General]
I updated my Amazon password in Italy
I went to update the credentials on the app where I confirmed the password and solved the anti-robot question
it gives me error 500
How do I solve it?

http://192.168.1.55:8123/auth/alexamedia/proxy/verify

Error

500 Internal Server Error Server got itself in trouble

**Comment by balajibalasa** [comment]
Same her any solutions will be helpfull

**Comment by codeclinic** [comment]
I have the same issue, no matter what I do.

Credentials and 2FA are all confirmed.

**Comment by pajaropinto** [comment]
Same problem also.

Y try with several fixed by google without success

**Comment by codeclinic** [comment]
I noticed in the debug logs of HA that the calls were being rejected because of authentication issues.

I don't know how to resolve that though.

**Comment by alphabravocharly72** [comment]
Same problem after new pw set

**Comment by zraken** [comment]
I ran into this problem today after updating to the latest alexa media player from HACS and a reboot. I was originally specifying a public facing DNS and when I switched to specifying the local dns address with valid SSL certificate (https) and it worked. I see that OP was using http, maybe there's a hard requirement for this to be https with valid certificate?
I still need to find out why my public DNS failed.

**Comment by Armand29** [comment]
Mine is totally broken as well after updating my Amazon password.  Errors out, cannot enable skill.

**Comment by bzh49** [comment]
same as Armand29 I Changed password internal error 500 instead of list of devices

**Comment by vinc1978** [comment]
Hi everyone. I've the same problem. I deleted the integration after 2024.8 problem. I can't reconfigure the integration.

**Reply by cy-bertrand** [reply]
try to rollback the integration to 4.12.8, it worked for me.

**Comment by BMRG-Holgat** [comment]
Was having the same problem, turned out it was the password I was using was not the correct one, as soon as this was corrected it worked a treat

**Comment by vectorr66** [comment]
I have never got the alexa media player to work with this same issue. 

![image](https://github.com/user-attachments/assets/7a00cf33-a870-4c5b-9932-1d8c538f89d1)

**Reply by skidazzit** [reply]
im getting the same error. did anyone figure out how to resolve this?

**Reply by Celestefano** [reply]
Same problem for me! News?

**Comment by xviti** [comment]
Hi all,
Has anyone solved this?

**Comment by JBecker32** [comment]
Same problem here, whatever I try, always Error 500.

**Comment by gshafer** [comment]
In the same boat. Has anyone got a solution for this problem?

**Comment by codeclinic** [comment]
I've not tested this yet but, I saw somewhere that the underlying library will now only work with 2FA .  As such it's impossible to log-in without 2FA on the account.

Give it a try and let us know if it works.
________________________________

**Comment by gshafer** [comment]
Phil —— Thanks for the tip. You are 100% correct. As soon as I enabled 2FA on the account, I was able to load the Alexa app. Thanks!

From: Phill Healey ***@***.***>
Date: Sunday, June 29, 2025 at 2:40 PM
To: alandtse/alexa_media_player ***@***.***>
Cc: gshafer ***@***.***>, Comment ***@***.***>
Subject: Re: [alandtse/alexa_media_player] http://192.168.1.55:8123/auth/alexamedia/proxy/verify 500 Internal Server Error Server got itself in trouble (Discussion #2133)

I've not tested this yet but, I saw somewhere that the underlying library will now only work with 2FA . As such it's impossible to log-in without 2FA on the account.

Give it a try and let us know if it works.
________________________________

**Comment by codeclinic** [comment]
Excellent. I'll have to get around to sorting mine out, now that I know it works.

Glad you got yours working.

________________________________

**Comment by qphoria** [comment]
Still can't get this working today. Always gives 500 error on the verify step. Account and pass are 100% correct and I've tried with and without 2SV

**Comment by Aipoc76** [comment]
cant get working, even added a app for authenticator.  Have tried both phone and app.  constantly get internal server error.

**Reply by danielbrunt57** [reply]
> cant get working, even added a app for authenticator. Have tried both phone and app. constantly get internal server error.

I had that happen once to me recently and it turned out I had accidentally entered an incorrect password for my Amazon account during that installation.  It manifested itself in the setup when the Amazon login window opened.  Selected Existing User then Login but then no OTP window opened after that, just a 500 error.  Did you see the Amazon OTP window and then get the 500 error or was there no OTP window and just the 500 error after user email + password?

**Reply by Aipoc76** [reply]
I've had both happen.  I can not figure out the rhyme or reason on what makes the difference.  Either way its still not working.

**Reply by danielbrunt57** [reply]
Have you tried using Chrome browser?  (It needs to be your default browser so AMP can open the page in it)
Also try clearing your browser's cache.

**Comment by Aipoc76** [comment]
got this from Chrome.  
<img width="646" height="721" alt="image" src="https://github.com/user-attachments/assets/6380504c-a748-4450-8aca-b1a306f164de" />


I normally use BRAVE, which is a offshoot of chrome.

**Comment by Aipoc76** [comment]
wonder if my firewall could be blocking something....  Will investigate more this afternoon.

**Comment by danielbrunt57** [comment]
Possibly.

**Comment by saviothecnic** [comment]
Confermo si deve disabilitare autentificzione a due fattori F2A dopo che si è sistemato su HA si puo anche riabilitare

<img width="630" height="690" alt="image" src="https://github.com/user-attachments/assets/1ea5b4d2-70de-4db8-9eb2-51fe3d857ae2" />
